platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -26,68 +28,105 @@
 
 public class CertificateCommand {
 
+    private static final String SAN_EXAMPLE = "-san DNS:localhost,IP:127.0.0.1";
+
     /**
      * Pass in a string to use as the common name of the certificate to be generated.
-     * Exception thrown if 0 arguments or more than 1 argument.
+     * Exception thrown if 0 arguments or more than 2 argument (or more than 4 when -san is used).
      *
      * @param args
      */
     public static void main(String args[]) {
-        if (args.length != 2) {
+        // try to extract -san if provided
+        int expected = 2;
+        String[] sans = null;
```

Comment:
now that the arguments are documented and since this is a local variable limited to this method, I think we are ok keeping it simple.